dev-environment/mlflow/Dockerfile

```diff
@@ -2,21 +2,20 @@ FROM conda/miniconda3:latest
 
 ENV MLFLOW_HOME /opt/mlflow
 ENV MLFLOW_WORK_DIR /home/mlflow
-ENV MLFLOW_VERSION 1.8.0
 ENV SERVER_PORT 5000
 ENV SERVER_HOST 0.0.0.0
 
 RUN apt-get update && apt-get install -y git
 
 RUN conda update conda
 RUN conda install -c anaconda jupyter=1.0.0
-RUN conda install -c conda-forge mlflow=1.8.0
 RUN conda install -c anaconda scikit-learn=0.22.1
 RUN conda install -c anaconda psycopg2=2.8.5
 RUN conda install -c anaconda boto3=1.14.12
-RUN conda install -c conda-forge azure-core=1.8.2
+RUN conda install -c conda-forge azure-core=1.9.0
 RUN conda install -c conda-forge azure-storage=0.36.0
-RUN conda install -c conda-forge azure-storage-blob=12.5.0
+RUN conda install -c conda-forge azure-storage-blob=12.6.0
```

Comment:
Why do we need Azure services? We don't keep anything in Azure Blob Storage.

Reply:
not sure, it was needed when doing the setup of environment, now only the version updated. Dataset is at http://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv

Reply:
Let's check if mflow works without these dependencies.

Reply:
no it doesn't. I had to add it manually (it's not an example copied from some webpage). Its needed to serve model using the MLmodel definition stored on S3